glean-core/src/internal_metrics.rs

```diff
@@ -10,6 +10,13 @@ pub struct CoreMetrics {
     pub first_run_date: DatetimeMetric,
     pub first_run_hour: DatetimeMetric,
     pub os: StringMetric,
+
+    /// The number of times we encountered an IO error
+    /// when writing a pending ping to disk.
+    ///
+    /// **Note**: Not a _core_ metric, but an error metric,
```

Comment:
I don't like it, but that's the only place I could reasonably add it without introducing yet another substructure or renaming stuff.

Reply:
We don't have it yet (it's a bit tricky, we will need a template that's different from the one for the RLB, because we actually need to generate for the _internal_ metric types, not the RLB types, also right now they are not global statics)